src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -322,6 +322,70 @@ function trapBubbledEventsLocal() {
         ),
       ];
       break;
+    case 'video':
+      inst._wrapperState.listeners = [
```

Comment:
I'm not entirely sure about this, but this is superfluent I believe. This is only for events that explicitly can emit events the instant the node is mounted into the DOM (and otherwise risk getting lost). Perhaps it applies to some of these `onLoadStart` (etc) perhaps, but I'm sure it does not apply to all of them.

@jimfb @zpao correct me if I'm wrong.